tests/unit/PaymentErrorDialog.spec.ts

```diff
@@ -122,10 +122,11 @@ describe('Payment Error Dialog', () => {
       {
         vuetify,
         store,
-        propsData: { dialog: true, warnings: [
-          { message: 'Test Warning 1' },
-          { message: 'Test Warning 2' }
-        ]}
+        propsData: { dialog: true,
+          warnings: [
+            { message: 'Test Warning 1' },
+            { message: 'Test Warning 2' }
+          ] }
```

Comment:
Lint fixes only in this file.

NB: these are auto-fixed if you do `npm run lint`.